test/cases/association_test.rb

```diff
@@ -43,8 +43,8 @@ def test_has_many
 
   def test_has_many_on_new_record
     Post.send(:has_many, :topics)
-    Topic.stubs(:find).returns([:unexpected_response])
-    assert_equal [], Post.new.topics.to_a
+
+    assert_kind_of ActiveResource::Collection, Post.new.topics
```

Comment:
This is now lazy loaded. So testing the same thing – that a network call isn't made